.github/workflows/issue_to_project.yml

```diff
@@ -23,3 +23,11 @@ jobs:
           project_id: 2
           resource_node_id: ${{ github.event.issue.node_id }}
           status_value: ${{ env.todo }} # Target status
+      - name: Add issue to Public Roadmap
+        uses: leonsteinhaeuser/project-beta-automations@v2.0.1
+        with:
+          gh_token: ${{ secrets.GH_CQ_BOT }}
+          organization: cloudquery
+          project_id: 7
+          resource_node_id: ${{ github.event.issue.node_id }}
+          status_value: No Status
```

Comment:
I'm not sure `No Status` will work as it has to be one of:
https://github.com/orgs/cloudquery/projects/7/settings/fields/Status
![image](https://user-images.githubusercontent.com/26760571/195347798-fa35c914-6f5f-44ad-b052-39863a7d6d30.png)

Unless I'm missing something and there's another status field.
How about we drop `status_value` as it's not required

Reply:
I am good with that